and-bible/app/src/main/java/net/bible/android/control/bookmark/BookmarkControl.java

```diff
@@ -12,6 +12,7 @@
 import net.bible.android.control.page.CurrentBiblePage;
 import net.bible.android.control.page.CurrentPageManager;
 import net.bible.android.control.page.window.ActiveWindowPageManagerProvider;
+import net.bible.android.control.versification.ConvertibleVerseRange;

```

Comment:
This is unused import

Reply:
@jsmitza thanks for this proposal and the ability to not only make a great suggestion but to also make the commitment to code a solution.  For me this is one of the key missing features that currently prevent me from using And Bible.  In reality it would be nice if there were separate features for "hilighting" and "bookmarking" but your idea is a fine proposal to at least let bookmarks function as hilights.